generator/types.go

```diff
@@ -651,6 +657,25 @@ func (f *Field) TypeSchemaName() string {
 	return parts[len(parts)-1]
 }
 
+func (f *Field) mustFindArrayLenght() int {
```

Comment:
`func arrayLen(f *Field) (int, error) {`

Not all fields are arrays, so it's not something that needs to be a method of field. Also, IMHO, `arrayLen` describes as well as `findArrayLength` what the function does, while being a lot more succint.